src/app/components/login/login.jade

```diff
@@ -0,0 +1,15 @@
+div(ng-controller="LoginController as loginCtrl")
+  h1.frontend-bootstrap-component-1
+    {{loginCtrl.loginPhrase}}
+  form(name='loginForm' ng-submit='loginForm.$valid && loginCtrl.login()' novalidate)
+    .input-control
+      | Username
+      input(name='usrname', ng-model='loginCtrl.usrname', required)
+    .input-control
+      | Password
+      input(name='pw1', ng-model='loginCtrl.password', type='password', required, ng-minlength="6")
+    .input-control
+      input(type='submit' value='Submit')			
+
+  a(ui-sref='home.index')
```

Comment:
Don't use a button. Redirect the user automatically in the success callback of the login